charts/bpa/templates/_helpers.tpl

```diff
@@ -93,10 +93,18 @@ generate hosts if not overriden
 generate ledger browser url
 */}}
 {{- define "bpa.ledgerBrowser" -}}
-{{- $ledgerBrowser := dict "bosch-test" "https://indy-test.bosch-digital.de" "idu" "https://explorer.idu.network" "bcovrin-test" "http://test.bcovrin.vonx.io" -}}
```

Comment:
Is that one outdated or should'nt be used, or is there a better option? I added that one to have another test option.

Reply:
not outdated, was just cleaning up any bcgov specific code from this repository, it just needs to reside in our fork.

Reply:
I think its ok to keep bcovrin-test as an option, basically any ledger that allows public registration can be added here.